vcluster/configure/vcluster-yaml/sync/to-host/advanced/custom-resources.mdx

```diff
@@ -41,20 +44,38 @@ sync:
         enabled: true
 ```
 
+## Versioned example
+
+To use a specific version of a CustomResourceDefinition (CRD), specify the version before the resource type in the `apiVersion` field, using the `/` separator. For example, to use version `v1` of the `certificates.cert-manager.io` CRD:
+
+```yaml
+sync:
+  toHost:
+    customResources:
+      certificates.cert-manager.io/v1:
+        enabled: true
+```
+
+:::info Unique CRD versions
+vCluster supports syncing only one version of a custom resource. If multiple versions are specified, the virtual cluster fails to start.
+
+If an upgrade introduces a version that is not yet synced, vCluster modifies the CRD inside the virtual cluster to include the new version.
+:::
 
 
```

Comment:
Same here, need to clarify somewhere above that version is required in host first. It looks like this doc is missing the original statement that the CRD itself also needs to pre-exist.

Reply:
Added